integration/spark/build.gradle

```diff
@@ -95,8 +95,9 @@ dependencies {
     testImplementation "org.testcontainers:junit-jupiter:${testcontainersVersion}"
     testImplementation "org.testcontainers:postgresql:${testcontainersVersion}"
     testImplementation "org.testcontainers:mockserver:${testcontainersVersion}"
+    testImplementation "org.testcontainers:kafka:1.16.0"
```

Comment:
We should use `${testcontainersVersion}`

Reply:
updated